WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -43,7 +43,9 @@
 
         <% if @past_competitions.length > 0 %>
           <div class="col-md-12" id="past">
-            <%= render 'index_table', competitions: @past_competitions, title: "Past competitions (last #{CompetitionsController::PAST_COMPETITIONS_DAYS} days)" %>
+            <% past_comps_title = "Past competitions" %>
+            <% past_comps_title += " (last #{CompetitionsController::PAST_COMPETITIONS_DAYS} days)" if params[:years] == "current"  %>
+            <%= render 'index_table', competitions: @past_competitions, title: past_comps_title %>
```

Comment:
This is great, but how do you feel about going the whole 9 yards? I'm thinking something like this:

```ruby
if params[:years] == "current"
  past_comps_title = "Competitions from the last #{CompetitionsController::PAST_COMPETITIONS_DAYS} days "
elsif params[:years] == "all"
  past_comps_title = "All past competitions EVARR"
else
  past_comps_title = "Competitions from #{params[:years]}"
```

The English will need a little professionalizing, but hopefully that illustrates my point.